gulpfile.js

```diff
@@ -343,6 +349,20 @@ function release_linux64() {
     return archive.finalize();
 }
 
+// Create distribution package for linux platform
+function release_linux32() {
+    var src = path.join(appsDir, pkg.name, 'linux64');
```

Comment:
I think that this is a bug (the third parameter I suppose that must be `linux32` and not `linux64`), but if that is true, when merging this PR: https://github.com/betaflight/betaflight-configurator/pull/775 then it will be not important, this function will be replaced by a generic one that solves this.